timescaledb/tutorials/analyze-intraday-stocks/fetch-and-ingest.md

```diff
@@ -0,0 +1,230 @@
+# 2. Fetch and insert intraday stock data
+
+In this step, you will
+
+* create a configuration file (optional)
+* fetch stock data
+* insert the data into TimescaleDB
+
+## Create a configuration file
+
+This is an optional step, but it is highly recommended not to store your password or other sensitive information
+directly in your code. Instead, create a configuration file, for example `config.py`, and include your 
+database connection details and Alpha Vantage API key in there:
+
+```python
+# example content of config.py:
+DB_USER = 'user'
+DB_PASS = 'passwd'
+DB_HOST = 'host'
+DB_PORT = '000'
+DB_NAME = 'db'
+APIKEY = 'alpha_vantage_apikey'
+```
+
+Later, whenever you need to reference any of the information from this config file, you just need to import it:
+```python
+import config
+apikey = config.APIKEY
+...
+```
+
+## Collect ticker symbols
+
+In order to fetch intraday stock data, you will need to know which ticker symbols you want to analyze.
+
+First, let's collect a list of symbols so later we can fetch their data!
+
+In general, you have a few options to gather a list of ticker symbols dynamically:
+
+* Scrape it from a public website (example code here).
+* Use an API that has this functionality.
+* Download it from an open repository
+
+To keep things simple, just download this CSV file to get started:
+
+* [Download top 100 US ticker symbols (based on market capitalization)]()
+
+## Read symbols from CSV file
+
+After downloading the CSV file into the project folder, read the symbols into a list:
+
+```python
+import csv
+with open('ticker_symbols.csv') as f:
+    reader = csv.reader(f)
+    symbols = [row[0] for row in reader]
+    print(symbols)
+
+['MMM', 'AOS', 'ABT', 'ABBV', 'ABMD']
+```
+
+Now you have a list of ticker symbols that you can use later to make requests to the Alpha Vantage API.
+
+## Fetching intraday stock data
+
+*It's not necessary to know how Alpha Vantage API works to complete this tutorial, so feel free to skip this section below, if you want.*
```

Comment:
```suggestion
*It's not necessary to know how Alpha Vantage API works to complete this tutorial, so feel free to skip this section, if you want.*
```

Generally try to avoid directional language, partly for accessibility, but also just because it won't always be true that the next bit is "below", it could be on a different page, or be moved entirely.